package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "eslint-config-extensionengine",
+  "version": "1.0.0",
+  "description": "Extension Engine's eslint config",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ExtensionEngine/eslint-config-extensionengine.git"
+  },
+  "scripts": {
+    "lint": "eslint ."
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "config",
+    "Extension Engine",
+    "javascript",
+    "vue",
+    "node"
+  ],
+  "author": "Zdravko Ćurić (https://github.com/zcuric)",
+  "contributors": [
+    {
+      "name": "Dario Vladović",
+      "url": "https://github.com/vladimyr"
+    },
+    {
+      "name": "Damir Perišić",
+      "url": "https://github.com/underscope"
+    },
+    {
+      "name": "Luka Šarić",
+      "url": "https://github.com/lukasaric"
+    },
+    {
+      "name": "Toma Zelić",
+      "url": "https://github.com/kronicker"
+    },
+    {
+      "name": "Zdravko Ćurić",
+      "url": "https://github.com/zcuric"
+    }
+  ],
```

Comment:
Remove this

Reply:
I'll remove this, but please in future CR's please provide the reasoning for removing, adding and changing stuff. Some things are understandable, but this isn't one of them.

I was thinking of giving credit to people that worked on defining this `eslint` config.  I guess we don't that. This was inspired by [`airbnb` approach](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json#L36).